modules/android/pages/vector-search.adoc

```diff
@@ -122,9 +181,19 @@ This metric focuses on the alignment of two vectors, the similarity of direction
 Only the direction of the vectors matter.
 The smaller the distance value, the more similar the vectors are.
 
+== Hybrid Vector Search
+
+You can use Vector Search with regular SQL++ query language. 
+Vector search will be performed on the documents that have been filtered based on the criteria specified in the WHERE clause.
+No LIMIT clause is required for Hybrid Vector Search.
+
+See the https://www.couchbase.com/blog/hybrid-search/[Hybrid Search blog post] for more information about Hybrid Search.
+
+IMPORTANT: You cannot `OR` with Hybrid Vector Search or when using `APPROX_VECTOR_DISTANCE()` with other expressions in the `WHERE` clause.
```

Comment:
I don't understand what you mean in the "important" note.

Reply:
Actioned, moved to working with page near relevant snippets for better context